### PR TITLE
Moving storage samples to py.test

### PR DESCRIPTION
**Pull request by jonparrott**
@jerjou - there was also a slight refactor to the resource thingy in `cloud.py`. Once this repository stabilizes, I'll like move a lot of this over to `gcp-python-repo-tools`.

### CODE REVIEW

File: testing/cloud.py [hunk 1]
```diff
@@ -47,6 +47,20 @@ def __getattr__(self, name):
         return os.environ[name]
 
 
+def get_resource_path(resource, local_path):
```
Comment (jerjou) [inline testing/cloud.py]: Any particular reason you didn't stick with the same method signature as `resource_path`? eg with the variable-length `*resource`?

Comment (jonparrott) [inline testing/cloud.py]: `local_path` makes things awkward.

Comment (jerjou) [inline testing/cloud.py]: Howso?

Comment (jonparrott) [inline testing/cloud.py]: Hmm. I guess I could make it the first argument if you want - that would more or less fix it.

Comment (jerjou) [inline testing/cloud.py]: Oh I see what the difference is. I didn't realize that you weren't *necessarily* getting the resource from the local path. I see where the awkwardness lies now.

Okay - it's fine the way it is. Suggestion if you like it:

    def get_resource_path(*resource, local_path=None):

and add a check for `local_path` before using it. That feels marginally more semantic for me, but whatever.

**Comment by jerjou** [comment]
LGTM

File: testing/cloud.py (hunk 1 above)
Comment (jonparrott) [inline testing/cloud.py]: You can do that with python 3, but not 2. It's invalid syntax to have vargs before kwargs.

Comment (jonparrott) [inline testing/cloud.py]: I'll likely refactor this a bit anyway when I lift it into python-repo-tools.

Comment (TRIMIXING1AI) [reply testing/cloud.py:50]: TWEAKING 🎼BEATS #TRIXMIXING1AI ♥️

Comment (TRIMIXING1AI) [reply testing/cloud.py:50]: TRIXMI@trixmixing1ai.com

**Pull request merged**